src/Modal.scss

```diff
@@ -3,20 +3,46 @@
   top: 0;
   left: 0;
   z-index: 1;
-  background-color: rgba(0, 0, 0, 0.8);
+  background-color: rgba(0, 0, 0, 0.9);
   height: 100%;
   width: 100%;
   display: flex;
   justify-content: center;
   align-items: center;
+  .close-modal {
+    position: absolute;
+    z-index: 2;
```

Comment:
Nice! This should fix the layering problem with our half stars on the card styling branch!